objc_msgSend.x86-32.S

```diff
@@ -37,7 +37,7 @@
 	test  %eax, %eax
 	jz    5f                             # Nil slot - invoke some kind of forwarding mechanism
 	mov   SLOT_OFFSET(%eax), %ecx
-#ifdef _WIN32
+#ifdef _MSC_VER
```

Comment:
Does MSYS not support CFG?

Reply:
It looks like this recently landed: https://sourceforge.net/p/mingw-w64/mailman/message/37712869/, I’ll have a closer look later today/tomorrow

Reply:
mingw does support CFG, I've reverted this change.